src/v2/Apps/Artwork/Components/SubmittedOrderModal/__tests__/SubmittedOrderModal.jest.tsx

```diff
@@ -0,0 +1,72 @@
+import { graphql } from "react-relay"
+import { setupTestWrapperTL } from "v2/DevTools/setupTestWrapper"
+import { screen, within, fireEvent, waitFor } from "@testing-library/react"
+import { SubmittedOrderModalFragmentContainer } from ".."
+
+jest.unmock("react-relay")
+
+const mockedResolver = {
+  CommerceOrderConnectionWithTotalCount: () => ({
+    edges: [{ node: { mode: "OFFER", stateExpiresAt: "Feb 28" } }],
+  }),
+}
+
+describe("SubmittedOrderModal", () => {
+  const { renderWithRelay } = setupTestWrapperTL({
+    Component: ({ me }) => (
```

Comment:
I don't understand why it's expecting to find `children` here:
```
Property 'me' does not exist on type '{ children?: ReactNode; }'
```
Afaik it's [sending all the props as props](https://github.com/artsy/force/blob/main/src/v2/DevTools/setupTestWrapper.tsx#L96-L103). Am I looking on the right place, though?

Reply:
this should do the work
```jsx
const { renderWithRelay } = setupTestWrapperTL<
    SubmittedOrderModal_Test_Query
  >({
    Component: props => (
      <SubmittedOrderModalFragmentContainer
        order={props!.me!.conversation!.orderConnection!.edges![0]!.node!}
      />
    ),
    ...
```
or if you don't want to be so type-specific, you could type the props as any after Component, will be less verbose and less noisy from TS workarounds